sr_ur_launch/launch/right_arm.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
```

Comment:
This file looks like a replacement for:

sr_ur_bringup/launch/sr_ur_right.launch

Should that one be deleted?

And the left arm one as well?

sr_ur_bringup/launch/sr_ur_left.launch